scripts/validate-flux-verify/main_test.go

```
@@ -0,0 +1,349 @@
+	}
+}
+
+// TestRealConfigValidates is acceptance criterion 2: the check must pass on the
+// config production actually deploys, not only on the fixtures above. It reads
+// the real file so a future edit that breaks verification fails this test.
+func TestRealConfigValidates(t *testing.T) {
+	t.Parallel()
+
+	path := filepath.Join("..", "..", "ksail.prod.yaml")
+
+	config, err := os.ReadFile(path)
```

golangci-lint (2.12.2)

[medium] G304: Potential file inclusion via variable

(gosec)